rpcs3/Emu/RSX/GL/GLOverlays.cpp

```diff
@@ -471,26 +471,59 @@ namespace gl
 			"uniform int limit_range;\n"
 			"uniform int stereo;\n"
 			"uniform int stereo_image_count;\n"
+			"uniform int stereo_display_method;\n"
 			"\n"
 			"vec4 read_source()\n"
 			"{\n"
 			"	if (stereo == 0) return texture(fs0, tc0);\n"
 			"\n"
-			"	vec4 left, right;\n"
+			"	vec4 left, right, final;\n"
 			"	if (stereo_image_count == 2)\n"
 			"	{\n"
-			"		left = texture(fs0, tc0);\n"
-			"		right = texture(fs1, tc0);\n"
+			"   vec2 coord_left = tc0 * vec2(1.f, 0.4898f);\n"
```

Comment:
The formatting in your added code is completely off.
The indentations are all over the place.
You're missing spaces in "if(" and after "//" in comments

Reply:
Indentation is weird, looks fine in vscode, however i think that's automatically accounting for the indentation already present, which i am struggling to figure out what it is.